cfpb/cfgov-refresh
# Add mega menu testing link option behind flag

Add another menu link toggle flag for IA usability testing this week on build.


## Changes

- Add menu link toggle in `mega-menu-for-ia-testing.html`

## Testing

- Check out branch
- Run gulp
- Create and activate an `IA_USER_TESTING_MENU` flag to see test version of Consumer Tools in mega menu

- Verify `Servicemembers` link
  - Verify that `Servicemembers and veterans` link in `Information for...` column goes to http://build.consumerfinance.gov/consumer-tools/servicemembers/
  - Create and activate an `IA_USER_TESTING_SV2` flag
  - Verify that `Servicemembers` link in `Information for...` column now goes to http://build.consumerfinance.gov/consumer-tools/servicemembers-2/

## Screenshots

<img width="1267" alt="screen shot 2017-05-11 at 10 41 05 am" src="https://cloud.githubusercontent.com/assets/778171/25963515/6440895e-3636-11e7-9e4f-1708ef37374b.png">
